### PR TITLE
removed redundant check and code cleanup

### PR DESCRIPTION
**Pull request by Madhu-1**

### CODE REVIEW

File: cmd/cluster-driver-registrar/k8s_register.go [hunk 1]
```diff
@@ -79,6 +69,14 @@ func kubernetesRegister(
 	}
 }
 
+func cleanup(clientSet *k8scsiclient.Clientset, csiDriver *k8scsi.CSIDriver) {
+	c := make(chan os.Signal, 1)
```
Comment (msau42) [inline cmd/cluster-driver-registrar/k8s_register.go]: This introduces a tiny race condition where the signal may not be setup before the process exits

Comment (Madhu-1) [reply cmd/cluster-driver-registrar/k8s_register.go]: @msau42  I moved the cleanup logic to a new function, will that introduces the race condition?

Comment (Madhu-1) [reply cmd/cluster-driver-registrar/k8s_register.go]: @msau42 yes it may introduce race , updated code PTAL

**Pull request merged**